mongo/check.py

```diff
@@ -862,7 +865,21 @@ def total_seconds(td):
         except KeyError:
             pass
 
-        dbnames = cli.database_names()
+        cmd = SON([("listDatabases", 1)])
```

Comment:
Can you add a comment with a link to the code in the mongo driver here and a comment with a reminder to fix this after the upgrade is done!